modules/dailymotionBidAdapter.js

```diff
@@ -30,7 +30,18 @@ export const spec = {
     method: 'POST',
     url: 'https://pb.dmxleo.com',
     data: {
-      bidder_request: bidderRequest,
+      bidder_request: {
+        gdprConsent: {
+          apiVersion: bidderRequest?.gdprConsent?.apiVersion || 1,
+          consentString: bidderRequest?.gdprConsent?.consentString || '',
+          // Cast boolean in any case (eg: if value is int) to ensure type
+          gdprApplies: !!bidderRequest?.gdprConsent?.gdprApplies,
```

Comment:
@sebmil-daily just to make sure, this would mean that `gdprApplies` would be `false` in case it is undefined

Reply:
Yes, that would keep the current processing logic.